File: src/main/java/com/max/guess/Game.java
package com.max.guess;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        Random random = new Random();
        int secret = random.nextInt(10)+1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret) {
            System.out.println("Enter a nember: ");
            number = scanner.nextInt();
            System.out.println(number);
        }
       /* int i=0;
        do {
            System.out.println(i);
            i++;
        } while (i < 3);*/
    }
}

File: src/main/kotlin/com/max/Game.kt
package com.max

import java.util.*

fun main(args: Array<String>) {
    val secret = Random().nextInt(10) +1;
    println(secret)
//    val scanner = Scanner(System.`in`)
    var number=0
    while(number != secret) {
        print("please guess a number: ")
//        number = scanner.nextInt()
        number = readLine()!!.toInt() //readline可能是null
        if (number < secret) {
            println("Guess higher")
        } else if (number > secret) {
            println("guess lower")
        } else {
            println("Great! the number is $number")
        }
    }
}